pkg/minishift/clusterup/clusterup.go

```diff
@@ -99,6 +100,7 @@ func ClusterUp(config *ClusterUpConfig, clusterUpParams map[string]string, runne
 		}
 	}
 
+        log.Debug(fmt.Sprintf("-- Running `oc` with: '%s'", strings.Join(cmdArgs, " ")))
```

Comment:
@thomasmckay extra space begin of this log.Debug.

@LalatenduMohanty @gbraad I can see the command args for oc cluster up when using below which I think what we anticipating here.

```
$ minishift start --show-libmachine-logs --v=5
[...]
About to run SSH command:
sudo mkdir -p /var/lib/minishift/openshift.local.config /var/lib/minishift/hostdata /var/lib/minishift/openshift.local.volumes /var/lib/minishift/openshift.local.pv
SSH cmd err, output: <nil>: 
-- OpenShift cluster will be configured with ...
   Version: v3.6.0
-- Checking `oc` support for startup flags ... 
   host-volumes-dir ... OK
   routing-suffix ... OK
   host-config-dir ... OK
   host-data-dir ... OK
   host-pv-dir ... OK
-- Running `oc` with: 'cluster up --use-existing-config --host-volumes-dir /var/lib/minishift/openshift.local.volumes --routing-suffix 192.168.42.79.nip.io --host-config-dir /var/lib/minishift/openshift.local.config --host-data-dir /var/lib/minishift/hostdata --host-pv-dir /var/lib/minishift/openshift.local.pv'
[...]
```

Reply:
@praveenkumar @thomasmckay Tried the PR and it looks good to me.